corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/bootstrap5/multifield.html

```diff
@@ -0,0 +1,7 @@
+{% load crispy_forms_field %}
+
+{% if field.is_hidden %}
+    {{ field }}
+{% else %}
+    {% crispy_field field %}  {# todo B5: check crispy #}
```

Comment:
I don't think this template needs to be split.

Reply:
Nevermind, this got reverted. It'd be helpful to run a rebase before creating the PR.